spec/formatting.md

```diff
@@ -284,7 +283,7 @@ the following steps are taken:
    _operand_ did not match that expected by the _function_,
    the _function_ SHOULD cause a _Bad Operand_ error to be emitted.
   
-   In all failure cases, use the _fallback value_ for the _expression_ as its _resolved value_.
+   In all failure cases, use a _fallback value_ as the _resolved value_ of the _expression_.
```

Comment:
I was going to comment on line 255 to the effect that "a _fallback value_" might want to be "the _fallback value_". Saying "the" is specific, even though the exact formulation of the fallback value is not (line 248 is different).

Reply:
On 248 the "the" is because we're re-using the operand's resolved value.

On 255 and 286 we need "a" because we need to create the value.

Is that ok, or is there a change here that you think ought to be made?

Reply:
I agree that 248 is different.

I guess the change would be necessary if we think that there will be a specific fallback value for every expression (that is, specific to that expression)--which I think is true. Using "a" implies you can use any old fallback value.......

But it's not that important.

Reply:
The fallback value is specific to the _expression_ with respect to the current _formatting context_. I think it's best to leave it as "a", but modify this to "return" instead of "use".

```suggestion
   In all failure cases, return a _fallback value_ as the _resolved value_ of the _expression_.
```